frontend/packages/dev-console/src/components/import/section/ResourceSection.scss

```diff
@@ -0,0 +1,11 @@
+.odc-resource-section__badge {
+  margin-left: var(--pf-global--spacer--md);
+}
+
+.odc-resource-section__option-text {
+  margin: var(--pf-global--spacer--md) 0;
+}
```

Comment:
Correct me if I am wrong here, but I am pretty sure we don't use this one anymore now that it is part of the RadioButton.

Reply:
removed the style definition